lib/stronger_parameters/constraints/integer_constraint.rb

```diff
@@ -6,7 +6,7 @@ class IntegerConstraint < Constraint
     def value(v)
       if v.is_a?(Integer)
         return v
-      elsif v.is_a?(String) && v =~ /\A-?\d+\Z/
+      elsif v.is_a?(String) && v =~ /\A-?(\d+|\s+)\d+\Z/
```

Comment:
This should probably be `/\A\s*-?\d+\Z/` to simply allow for leading whitespace. `"  -42".to_i # => -42`

The alternative is to call `.strip` on v (including on the return line below).